xinference/deploy/cmdline.py

```diff
@@ -212,13 +287,20 @@ def unregister_model(
     )
 
 
-@cli.command("registrations")
+@cli.command("registrations", help="Lists all registered models in Xinference.")
 @click.option(
     "--endpoint",
     "-e",
     type=str,
+    help="Endpoint URL to retrieve the list of registrations.",
```

Comment:
```suggestion
    help="Xinference endpoint.",
```

Use the help messages above for all `endpoint` options.